packages/devtools_app/lib/src/network/flutter/http_cookies_tab.dart

```diff
@@ -0,0 +1,124 @@
+        child: Text(
+          title,
+          style: _headerTextStyle,
+          overflow: TextOverflow.fade,
+        ),
+      ),
+      numeric: numeric);
+
+  DataRow _buildRow(int index, Cookie cookie, {bool requestCookies = false}) =>
+      DataRow.byIndex(index: index, cells: <DataCell>[
+        _buildCell(cookie.name),
+        _buildCell(cookie.value),
+        if (!requestCookies) ...[
+          _buildCell(cookie.domain ?? 'N/A'),
+          _buildCell(cookie.path ?? 'N/A'),
+          _buildCell(cookie.expires?.toString() ?? 'N/A'),
+          _buildCell(cookie.value.length.toString()),
+          _buildIconCell(!cookie.httpOnly ? Icons.check : Icons.close),
+          _buildIconCell(!cookie.secure ? Icons.check : Icons.close),
+        ]
+      ]);
+
+  DataCell _buildCell(String value) => DataCell(Text(value));
+  DataCell _buildIconCell(IconData icon) => DataCell(Icon(icon));
+
+  Widget _buildCookiesTable(
+      String title, List<Cookie> cookies, BoxConstraints constraints,
+      {bool requestCookies = false}) {
+    return Column(
+      children: <Widget>[
+        Align(
+          alignment: Alignment.centerLeft,
+          child: Text(
+            title,
+            style: const TextStyle(
+                color: Colors.lightBlue,
+                fontSize: 18,
+                fontWeight: FontWeight.bold),
+          ),
+        ),
+        Padding(
+          padding: const EdgeInsets.only(bottom: 24.0),
+          child: Align(
+            alignment: Alignment.centerLeft,
+            child: SingleChildScrollView(
+              scrollDirection: Axis.horizontal,
+              child: ConstrainedBox(
+                constraints: requestCookies
+                    ? const BoxConstraints()
+                    : BoxConstraints(minWidth: constraints.minWidth),
+                child: DataTable(
+                  columns: [
+                    _buildColumn('Name'),
+                    _buildColumn('Value'),
+                    if (!requestCookies) ...[
+                      _buildColumn('Domain'),
+                      _buildColumn('Path'),
+                      _buildColumn('Expires / Max Age'),
+                      _buildColumn('Size', numeric: true),
+                      _buildColumn('HttpOnly'),
+                      _buildColumn('Secure'),
+                    ]
+                  ],
+                  rows: [
+                    for (int i = 0; i < cookies.length; ++i)
+                      _buildRow(i, cookies[i], requestCookies: requestCookies)
+                  ],
+                ),
+              ),
+            ),
+          ),
+        ),
+      ],
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final requestCookies = data.requestCookies;
+    final responseCookies = data.responseCookies;
+    return (requestCookies.isEmpty && responseCookies.isEmpty)
+        ? const Center(
+            child: Text('No Cookies', style: TextStyle(fontSize: 20)))
```

Comment:
prefer using a smaller # of unique fonts. Is there a reason this shouldn't also be 18 and be controlled by the theme?

Reply:
Most of this was me hacking away and trying to get something that looked good. I've changed most of these TextStyles to use the app theming.